src/main/java/seedu/duke/Ui.java

```diff
@@ -277,6 +311,14 @@ public void displayNoOfClients(int i) {
         }
     }
 
+    public void displayNoOfPairs(int i) {
+        if (i == 1) {
```

Comment:
Can consider changing the condition to a boolean variable with meaningful name so that when a user read the code, it will be more intuitive